src/Tasks/Microsoft.NET.Build.Tasks/RunReadyToRunCompiler.cs

```diff
@@ -103,12 +151,14 @@ protected override bool ValidateParameters()
 
                 if (!String.IsNullOrEmpty(DiaSymReader) && !File.Exists(DiaSymReader))
                 {
+                    Log.LogError(Strings.DiaSymReaderLibraryNotFound, DiaSymReader);
                     return false;
                 }
 
                 // R2R image has to be created before emitting native symbols (crossgen needs this as an input argument)
                 if (String.IsNullOrEmpty(_outputPDBImage) || !File.Exists(_outputR2RImage))
                 {
+                    Log.LogError(Strings.PDBGeneratorInputExecutableNotFound, _outputR2RImage);
```

Comment:
This error message would be confusing in case the R2R image is specified correctly, but `_outputPDBImage` is `null`.

Reply:
Good point, I have split this into two conditions with two error messages.